doc/manuals/admin/database_model.md

```diff
@@ -343,6 +343,7 @@ notifications. It is a number between 0 and 1800000. If defined to 0 or omitted,
 -   **covered**: a boolean field that specifies if all `attrs` have to be included in notifications (if value is true)
     or only the ones existing in the triggering entity (if value is false or field is omitted).
     More information in [covered subscription section in Orion API specification](../orion-api.md#covered-subscriptions).
+-   **notifyOnMetadataChange**: if `true` metadata is considered part of the value of the attribute regarding subscription triggering. If `false` metadata is not considered part of the value of the attribute regarding subscription triggering. Default behaviour (if omitted) is the one for `true`.
```

Comment:
Should this new field be added to API.md?

Reply:
It is, in this separate PR https://github.com/telefonicaid/fiware-orion/pull/4308

Reply:
Ok, NTC